examples/dreambooth/train_dreambooth_lora_flux.py

```diff
@@ -1372,7 +1372,6 @@ def load_model_hook(models, input_dir):
             # changes the learning rate of text_encoder_parameters_one and text_encoder_parameters_two to be
             # --learning_rate
             params_to_optimize[1]["lr"] = args.learning_rate
-            params_to_optimize[2]["lr"] = args.learning_rate
```

Comment:
I am not sure if this is the right fix because what happens when we train the text encoder, too? Cc @linoytsaban

Reply:
when we enable the option for training text encoders too, we only target the clip model and not the T5 for flux, hence this line was unnecessary from what I noticed

Reply:
hence also the reason for the indexerror^

Reply:
indeed for now we support full fine-tuning of the CLIP encoder only when `--train_text_encoder` is enabled, so simply removing this line for now should be good!